init/eessi_archdetect.sh

```diff
@@ -150,8 +150,25 @@ cpupath(){
     fi
 }
 
+accelpath() {
+    # If EESSI_ACCELERATOR_TARGET_OVERRIDE is set, use it
+    log "DEBUG" "accelpath: Override variable set as '$EESSI_ACCELERATOR_TARGET_OVERRIDE' "
```

Comment:
What about sanity checking this a bit? We know that it should start with `accel/nvidia/cc`

Reply:
Yeah, that makes sense, I'll tackle that